lib/modules/platform/gerrit/index.ts

```diff
@@ -44,7 +45,10 @@ export const id = 'gerrit';
 
 const defaults: {
```

Comment:
can we move this to common or utils, so client can directly access it? so we don't need to pass the version to each client call.